### ISSUE TITLE
generate video in the inference

### ISSUE THREAD

**Issue by SophieZhou** [esdolo/FreeVS#11]
Hi, Sorry to bother you again. 
I am not sure about some details in the paper.  When you generate the video with 8 frames, the pseudo-images should be 8 frames as input?

**Comment by esdolo** [comment]
Yes.

**Comment by SophieZhou** [comment]
> Yes.

Thanks

**Comment by SophieZhou** [comment]
Sorry to bother you again. During inference, the text messages are generated by the first frame of each 8 frames video, is it right? I am not sure how to generate long video  on your project page.

**Comment by esdolo** [comment]
Yes. Now the code is released and you can check the inference script.
